net/haproxy/Makefile

```diff
@@ -1,8 +1,8 @@
 PLUGIN_NAME=		haproxy
 PLUGIN_VERSION=		2.15
-PLUGIN_REVISION=	1
+PLUGIN_REVISION=	2
```

Comment:
```suggestion
PLUGIN_REVISION=	1
```

no need for this when dependency changes

Reply:
Fixed.